### DISCUSSION TITLE
Configured bigwig peaks do not appear

### DISCUSSION THREAD

**Discussion by ouyang-lang** [GMOD/jbrowse-components#3141] [Q&A]
Hello, I am configuring the bigwig file on jbrowse2 recently, but I have configured bigwig according to the configuration in the document, but its peak does not appear, and I do not know why。

my tracks.js code：
```json
{
    "type": "MultiQuantitativeTrack",
    "trackId": "microarray_multi",
    "name": "MultiWig",
    "category": ["ENCODE bigWigs"],
    "assemblyNames": ["GRCm38"],
    "adapter": {
      "type": "MultiWiggleAdapter",
      "bigWigs": [
        "https://www.encodeproject.org/files/ENCFF055ZII/@@download/ENCFF055ZII.bigWig",
        "https://www.encodeproject.org/files/ENCFF826HEW/@@download/ENCFF826HEW.bigWig",
        "https://www.encodeproject.org/files/ENCFF858LIM/@@download/ENCFF858LIM.bigWig",
        "https://www.encodeproject.org/files/ENCFF425TNW/@@download/ENCFF425TNW.bigWig",
        "https://www.encodeproject.org/files/ENCFF207RBY/@@download/ENCFF207RBY.bigWig",
        "https://www.encodeproject.org/files/ENCFF289CTN/@@download/ENCFF289CTN.bigWig",
        "https://www.encodeproject.org/files/ENCFF884IEG/@@download/ENCFF884IEG.bigWig",
        "https://www.encodeproject.org/files/ENCFF495SBQ/@@download/ENCFF495SBQ.bigWig",
        "https://www.encodeproject.org/files/ENCFF959EZF/@@download/ENCFF959EZF.bigWig",
        "https://www.encodeproject.org/files/ENCFF926YZX/@@download/ENCFF926YZX.bigWig",
        "https://www.encodeproject.org/files/ENCFF269CHA/@@download/ENCFF269CHA.bigWig",
        "https://www.encodeproject.org/files/ENCFF857KTJ/@@download/ENCFF857KTJ.bigWig",
        "https://www.encodeproject.org/files/ENCFF109KCQ/@@download/ENCFF109KCQ.bigWig",
        "https://www.encodeproject.org/files/ENCFF942TZX/@@download/ENCFF942TZX.bigWig",
        "https://www.encodeproject.org/files/ENCFF140HPM/@@download/ENCFF140HPM.bigWig",
        "https://www.encodeproject.org/files/ENCFF305JRR/@@download/ENCFF305JRR.bigWig",
        "https://www.encodeproject.org/files/ENCFF739FDJ/@@download/ENCFF739FDJ.bigWig",
        "https://www.encodeproject.org/files/ENCFF518OJP/@@download/ENCFF518OJP.bigWig",
        "https://www.encodeproject.org/files/ENCFF810HHS/@@download/ENCFF810HHS.bigWig",
        "https://www.encodeproject.org/files/ENCFF939JSB/@@download/ENCFF939JSB.bigWig",
        "https://www.encodeproject.org/files/ENCFF041TAK/@@download/ENCFF041TAK.bigWig"
      ]
    }
```

my picture：
![微信图片_20220816222227](https://user-images.githubusercontent.com/81238855/184903867-dc23ed95-1f73-4946-92ee-3d62f082d617.jpg)

**Comment by garrettjstevens** [comment]
This is most likely due to mismatching chromosome names between the BigWigs and your reference sequence. For example, your reference sequence may use `19` while your BigWig uses `chr19`. You can read about configuring aliasing between the two reference names here: https://jbrowse.org/jb2/docs/config_guide/#configuring-reference-name-aliasing

**Reply by ouyang-lang** [reply]
> 这很可能是由于 BigWigs 和您的参考序列之间的染色体名称不匹配。例如，您的参考序列可能使用`19`，而您的 BigWig 使用`chr19`. 您可以在此处阅读有关在两个参考名称之间配置别名的信息：[https ://jbrowse.org/jb2/docs/config_guide/#configuring-reference-name-aliasing](https://jbrowse.org/jb2/docs/config_guide/#configuring-reference-name-aliasing)

Even if I use alias, it's still the same no peaks are displayed, my chromosome alias format
![image](https://user-images.githubusercontent.com/81238855/185005736-5f3c0af8-0ade-479b-bd22-59b2d558c101.png)

**Reply by garrettjstevens** [reply]
You may need to switch the order of the columns, i.e. have `1` in the first column and `chr1` in the second column. The first column has to match what's in your reference sequence.

**Reply by ouyang-lang** [reply]
> 您可能需要切换列的顺序，即`1`在第一列和`chr1`第二列中。第一列必须与您的参考序列中的内容相匹配。

Thank you very much for your reply, I have tried changing the order of the first column and the second column, and then my bigwig file can show peaks, but it reports this error now, I don't know what the error means?

error：
compositeMap.ts:17 Uncaught TypeError: submap.has is not a function
![image](https://user-images.githubusercontent.com/81238855/185014631-4bd3f3c5-13b2-4304-ad23-c918e889b8c0.png)

**Comment by garrettjstevens** [comment]
I'm not sure what that means, I'll have to look into it.

**Reply by ouyang-lang** [reply]
> 我不确定这意味着什么，我将不得不调查一下。

Ok, thank you, if you investigate clearly, look forward to your reply

**Reply by garrettjstevens** [reply]
This is now tracked as a bug here: https://github.com/GMOD/jbrowse-components/issues/3155

**Reply by cmdcolin** [reply]
this submap.has issue was fixed in 2.1.2 just for reference :)
